bundles/binding/org.openhab.binding.fatekplc/src/main/resources/jfatek/SERIALFatekConnectionFactory

```diff
@@ -0,0 +1 @@
+org.openhab.binding.fatekplc.serial.SerialFatekConnectionFactory
```

Comment:
This resource contains name of class which is responsible for implementing serial transport for JFatek library.

As you see Serial transort for this binding is not implemeted yet.
I only add skeleton class which must be completed to support this trasport.